src/main/resources/templates/ccda/ccda.ftl

```diff
@@ -132,11 +132,6 @@
       <#if ehr_careplans?has_content>
         <#include "care_goals.ftl">
       </#if>
-      <#if ehr_imaging_studies?has_content>
-        <#include "diagnostic_imaging_reports.ftl">
-      <#else>
-        <#include "diagnostic_imaging_reports_no_current.ftl" parse=false>
-      </#if>
```

Comment:
So, we're no longer using the `diagnostic_imaging*.ftl` templates at all?

Maybe we should just remove them?

Reply:
I'll remove those templates.

I removed them from CCDA output because the diagnostic imaging section is not one that is included in the CCD document template that we are using. The document templates list mandatory and optional sections. I wasn't able to easily find an answer as to whether including a section template outside of those listed was valid.